buster/documents_manager/deeplake.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from typing import Optional
+
+import openai
+import pandas as pd
+
+from buster.utils import zip_contents
+
+from .base import REQUIRED_COLUMNS, DocumentsManager, get_embedding_openai
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def read_csv(filename: str):
```

Comment:
unused? And weird place to put it in any case

Reply:
removed